doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1791,6 +1791,12 @@ references to automatically generated names.
 
    This :term:`option` specifies the prefix to use when generating names.
 
+.. flag:: Mangle Names Light
+
+   When this :term:`flag` is set (it is off by default),
+   the behaviour of :flag:`Mangle Names` is changed to only prepend
+   the :opt:`Mangle Names Prefix` to the original name.
```

Comment:
```suggestion
   When this :term:`flag` is set (it is off by default),
   the names generated by :flag:`Mangle Names` only add
   the :opt:`Mangle Names Prefix` to the original name.
```

Since it's a "prefix", "prepend" is not needed.  IMHO.